subsys/compress/Kconfig

```diff
@@ -0,0 +1,100 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config NRF_COMPRESS
+	bool "nRF compression/decompression library"
```

Comment:
Was there a specific request from stakeholders to have this library named with `nRF`? That's how nRF is usually added.

Reply:
Dropped nRF